apps/console/src/features/identity-providers/components/settings/authenticator-settings.tsx

```diff
@@ -26,11 +26,15 @@ import {
     SegmentedAccordionTitleActionInterface
 } from "@wso2is/react-components";
 import _ from "lodash";
-import React, { FormEvent, FunctionComponent, MouseEvent, ReactElement, useEffect, useState } from "react";
+import React, { FormEvent, FunctionComponent, MouseEvent, ReactElement, SyntheticEvent, useEffect, useState } from "react";
```

Comment:
Possible line length exceeding. Stick to 120 characters per line. Check other instances as well.